corehq/apps/app_manager/templates/app_manager/partials/forms/case_config_advanced.html

```diff
@@ -190,13 +190,18 @@ <h4 class="panel-title panel-title-nolink">
 
 <script type="text/html" id="case-config:case-action">
     <div class="panel-heading clickable panel-heading-case-action" data-toggle="collapse"
-         data-bind="attr: {href: '#' + actionType + $index()}">
+         data-bind="attr: {href: '#' + actionType + $index()}, css: {'not-droppable': moveable()}">
         <h4 class="panel-title panel-title-nolink" data-bind="html: header"></h4>
     </div>
-    <button class="case-action-remove btn btn-purple"
-            data-bind="openModal: 'remove-action-modal-template', visible: true">
-        <i class="fa fa-trash"></i>
-    </button>
+    <div class="panel-case-actions-actions">
+        <a href="#" role="button" class="case-action-move btn btn-purple grip" data-bind="visible: moveable()">
+            <i class="fa fa-arrows-v"></i>
+        </a>
+        <button class="case-action-remove btn btn-purple"
+                data-bind="openModal: 'remove-action-modal-template', visible: true">
```

Comment:
Realize you didn't introduce this, but do you have any idea what `visible: true` is for?

Reply:
not sure. appears to work without that https://github.com/dimagi/commcare-hq/pull/23689/commits/f801534b884d56b01ed988ffd5746e470c1bda67